src/gc.cpp

```diff
@@ -243,6 +243,8 @@ NO_SANITIZE_ADDRESS void Heap::scan_memory(Cell::Visitor &visitor, void *start,
         Cell *potential_cell = *reinterpret_cast<Cell **>(ptr); // NOLINT
         if (!potential_cell)
             continue;
+        if ((reinterpret_cast<uintptr_t>(potential_cell) & 0b11) != 0b00)
```

Comment:
We're 8 bit aligned (or rather: the OS is 8 bit aligned), so I would guess we could change this to `& 0b111`.

Reply:
Yeah I had that before and the `asan` test failed with an unexpected object collection, so I'm trying this to see what's up. Just experimenting. :D

Reply:
...and it's still failing, so I guess there's some other issue. Probably the range check. 🤔